frontend/pages/index.page.tsx

```diff
@@ -59,111 +89,66 @@ export default function Home() {
 
   return (
     <Layout>
-      <Grid container spacing={2} sx={{ p: 4 }}>
-        <Grid item xs={12}>
-          <Container>
-            <Box
-              sx={{
-                flex: "1 1",
-                display: "flex",
-                alignItems: "center",
-                justifyContent: "center",
-                margin: "1.5rem 1rem 1.5rem 1rem",
-              }}
-            >
-              <Card elevation={5}>
-                <CardContent>
-                  <Container>
-                    <p>Search modules by topic, entity, or keywords.</p>
-                    {searchForm}
-                  </Container>
-                </CardContent>
-              </Card>
-            </Box>
-          </Container>
-        </Grid>
-      </Grid>
-      <Grid
-        container
-        spacing={{ xs: 2, md: 3 }}
-        columns={{ xs: 12 }}
-        alignItems={"center"}
-        justifyContent={"center"}
-      >
-        <Grid item xs={12} sm={6} md={6} alignItems="center" justifyContent="center">
-          <Container>
+      <Grid container justifyContent="center" spacing={{ xs: 2, md: 3, lg: 4 }} sx={{ p: 4 }}>
+        <GridItem>
+          <StyledCard elevation={5} sx={{ maxWidth: { lg: "50rem" } }}>
+            <StyledCardHeader title="About ModularHistory" />
             <AboutModularHistory />
-          </Container>
-        </Grid>
-        <Grid item xs={12} sm={6} md={6}>
-          <Container>
-            <FeaturedContent />
-          </Container>
-        </Grid>
-      </Grid>
-      <Grid container spacing={2} sx={{ p: 4 }}>
-        <Grid item xs={12}>
-          <Card elevation={5}>
+          </StyledCard>
+        </GridItem>
+        <GridItem>
+          <StyledCard elevation={5}>
+            <StyledCardHeader title="Module Search" />
```

Comment:
This is what `raised` looks like (only applied to "Module Search" card).

![image](https://user-images.githubusercontent.com/24942155/140573822-7a526176-ff59-4fd3-bab1-11d506805249.png)